interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -559,8 +559,19 @@ public void clientStreaming() throws Exception {
       requestObserver.onNext(request);
     }
     requestObserver.onCompleted();
-    assertEquals(goldenResponse, responseObserver.firstValue().get());
     responseObserver.awaitCompletion();
+    assertEquals(goldenResponse, responseObserver.firstValue().get());
+    assertThat(responseObserver.getValues()).hasSize(1);
+    final Throwable t = responseObserver.getError();
+    if (t != null) {
+      // AssertionError(String, Throwable) only present in Android API 19+ & Java 7+
```

Comment:
Can we just use `new RuntimeException(t)` instead? Or maybe even `assertNull(t)`.

Reply:
Now I use `new AssertionError(t)` instead. 
`assertNull(t)` seems not to show details of `t` clear enough.